src/west/manifest.py

```diff
@@ -278,7 +278,7 @@ def __init__(self, name, remote, defaults, path=None, clone_depth=None,
         self.remote = remote
         self.url = remote.url + '/' + name
         self.path = path or name
-        self.abspath = os.path.join(util.west_topdir(), self.path)
+        self.abspath = os.path.join(util.west_topdir(), os.path.normpath(self.path))
```

Comment:
Is this right? Why isn't it `os.path.normpath(os.path.join(util.west_topdir(), self.path))`?

Reply:
Will fix